circleci/integration-test

```diff
@@ -27,7 +27,8 @@ TESTING_SERVICE_USER=$2
 TESTING_SERVICE_PASS=$3
 APP_NAME=$4
 # TESTS_TO_RUN gets a default value, if not specified
-TESTS_TO_RUN=${5:-latest}
+# It is a JSON value whose schema corresponds with our internal testing spec schema.
+TESTS_TO_RUN=${5:-"[{\"type\": \"launchpad\"}]"}
```

Comment:
once this is merged can the "default" case be removed? https://github.com/Clever/circle-ci-integrations/pull/135/files/5cccfe68c0896f27488ac3fabbd19a66668b328d..42b6efa4222f0b404d3a2a3269630c614a4fedeb#diff-b244dd43c3bffd7fbbcb923b02909ff042a0729888527e0fa833ec2aa61c30ccR58

Reply:
Yes, I think so, I'll make that as a separate PR once this seem to be working.